src/pool/Pool.t.sol

```diff
@@ -1221,6 +1231,169 @@ contract PoolTest is BCdpManagerTestBase {
         pool.emergencyExecute(address(d), data);
     }
 
+    function testSetDaiToUsdPriceFeed() public {
+        address oldPriceFeed = address(pool.dai2usd());
```

Comment:
addition check here should be about the actual value.
set different price and see that it return different price